all.json

```diff
@@ -15,6 +15,44 @@
 		"1inch-wallet.org",
 		"1inch.tk",
 		"1metamask.com",
+		"accountvalidation.pages.dev",
+		"azukis.art",
+		"bayc-land.mints-nft.live",
+		"bnbfaucet.site",
+		"bonowatch.com",
+		"coinfactory.digital",
+		"comicsapesocialsclub.com",
+		"corriectionbot.co",
+		"cryptolightinvestment.com",
+		"eth-airdrops.io",
+		"faucetlists.ga",
```

Comment:
This one just seems to list faucets? Clicked through to a couple and they are external and _seems_ legit?

Reply:
the .ga threw me off. freenom.
very little legit freenom sites overall (close to below 1%)